src/app/community-view/community-invitation-dialog.component.ts

```diff
@@ -86,10 +97,29 @@ export class CommunityInvitationDialogComponent implements OnInit {
       filter(search => typeof search === 'string'),
       debounceTime(500),
       distinctUntilChanged(),
-      switchMap(search => this.usersService.getCommunityUserSuggestions(this.dialogData.communityId, search))
+      switchMap(search => this.communityUserService.getCommunityUserSuggestions(this.dialogData.communityId, search))
     );
   }
 
+  private loadRecommendedUsers() {
+    this.communityUserService.getRecommendedUsers(this.dialogData.communityId)
+      .subscribe(users => {
+        this.recommendedUsers = users;
+      }, errorResponse => {
+        this.handleErrorResponse(errorResponse);
+      });
+  }
+
+  private handleErrorResponse(errorResponse) {
+    this.errorMessage = this.globalErrorHandlerService.createFullMessage(errorResponse);
+    // Dirty fix because of: https://github.com/angular/angular/issues/17772
+    this.changeDetector.markForCheck();
+  }
+
+  private existsInUsersArray(user: User): boolean {
+    return this.usersArray.find(item => item.id === user.id) != null;
```

Comment:
It would be good to replace `find` with `some` to avoid iteration over the array as soon as the element is found.